### PR TITLE
Changed title attribute to be translatable

### PR DESCRIPTION
**Pull request by Dillon-Brown**
## Description

As described by @horus68 in #4673

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [X] My code follows the code style of this project found [here](https://suitecrm.com/wiki/index.php/Coding_Standards).
- [ ] My change requires a change to the documentation.
- [X] I have read the [**How to Contribute**](https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM) guidelines.

### CODE REVIEW

File: include/language/en_us.lang.php [hunk 1]
```diff
@@ -2439,6 +2439,7 @@
     'LBL_OK' => 'Ok',
 
     'LBL_COLUMNS_FILTER_HEADER_TITLE' => 'Choose columns',
```
Comment (Mausino) [inline include/language/en_us.lang.php]: There is issue, it should be here word  "Filter"

Comment (daniel-samson) [reply include/language/en_us.lang.php]: @Mausino  see 'LBL_FILTER_HEADER_TITLE' => 'Filter' . 'LBL_FILTER_HEADER_TITLE' is being used for the filter icon. The columns 'LBL_COLUMN_CHOSER' is used for the column filter.

Comment (Mausino) [reply include/language/en_us.lang.php]: @Dillon-Brown thank you. yes it was my mistake. sorry

**Comment by pgorod** [comment]
The word is `chooser`, not `choser`, unless me (and my dictionary) are mistaken.

**Pull request merged**